tests/cli_test.py

```diff
@@ -90,7 +97,7 @@ def test_simple(runner):
 
     result = runner.invoke(main_khal)
     assert not result.exception
-    assert result.output == 'No events\n'
+    assert strip_ansi_escapes(result.output) == 'No events\n'
```

Comment:
It is -- it strips colorcodes from the given string if its running under click's test runner.